README.md

```diff
@@ -95,6 +95,19 @@ contributors.
   needed.
 - Add a `replace` directive at the root `go.mod` file so your component is
   included in the build of the contrib executable.
+  
+### How to get proper reviews?
+
+We recommend that any PR (unless it is trivial) to be smaller than
+500 lines (excluding go mod/sum changes).
+When adding a new component consider submitting different PRs for:
+
+* First PR should include the overall structure of the new component:
+  * Readme, configuration, and factory implementation usually using the helper factory structs.
+  * This PR is usually trivial to review, so the size limit does not apply to it.
+* Second PR should include the concrete implementation of the component.
+If the size of this PR is larger than the recommended size consider splitting it in multiple PRs.
+* Last PR should be to enable the new component and add it to the `otelcontribcol` binary.
```

Comment:
I suggest also adding: "The component should be enabled only after sufficient tests and code coverage are attained and there is enough confidence in the stability and quality of the component."

Reply:
Should also specify what "enabling" the component means. Do you mean adding it to `components.go`?

Reply:
Maybe a note about refactoring? I'm assuming refactoring should generally go into a separate PR?